CMakeLists.txt

```diff
@@ -435,6 +435,15 @@ if(BUILD_TESTS)
   )
   set_property(TEST raft_scenario_test PROPERTY LABELS raft_scenario)
 
+  # Add and remove nodes
```

Comment:
The reconfiguration and code_update tests spin up many nodes which can slow down the tests dramatically(*). Moving this earlier so that it _should_ get executed earlier, when less expensive tests are running.

(*):
```
      Start 47: reconfiguration_test
36/40 Test #46: schema_tests .....................   Passed    6.74 sec
      Start 48: code_update_test
37/40 Test #47: reconfiguration_test .............***Failed  194.51 sec
      Start 50: end_to_end_logging_raft
38/40 Test #50: end_to_end_logging_raft ..........   Passed   33.98 sec
      Start 51: election_tests_raft
39/40 Test #48: code_update_test .................***Timeout 240.00 sec
40/40 Test #51: election_tests_raft ..............   Passed   33.41 sec

```